test/framework/options.py

```diff
@@ -2350,6 +2350,34 @@ def test_new_update_pr(self):
             regex = re.compile(regex, re.M)
             self.assertTrue(regex.search(txt), "Pattern '%s' found in: %s" % (regex.pattern, txt))
 
+    def test_empty_pr(self):
+        """Test use of --new-pr (dry run only) with no changes"""
+        if self.github_token is None:
+            print "Skipping test_new_pr, no GitHub token available?"
+            return
+
+        # get file from develop branch
+        test_ecs_dir = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'easyconfigs')
```

Comment:
download to `self.test_prefix`, that gets cleaned up automagically

otherwise if this test fails, the downloaded easyconfigs will stick around in `test/framework/easyconfigs` subdir